maven-reactor-projects/maven-reactor-projects.groovy

```diff
@@ -0,0 +1,7 @@
+/**
+ * This Groovy script adds a custom value for each Maven project in the reactor.
+ */
+
+def buildScan = session.lookup('com.gradle.maven.extension.api.scan.BuildScanApi')
+
+buildScan.value('project', project.name)
```

Comment:
❌ This will override the value for every project. Sorry I missed that, it was also true for the earlier version of the script.